lib/apiservers/engine/backends/archive.go

```diff
@@ -447,7 +442,7 @@ func (wm *ArchiveStreamWriterMap) WriterForAsset(proxy proxy.VicArchiveProxy, ci
 			deviceID = aw.mountPoint.Name
 			store = constants.VolumeStoreName
 		}
-		rawWriter, err := proxy.ArchiveImportWriter(wm.op, store, deviceID, aw.filterSpec)
+		rawWriter, err := proxy.ArchiveImportWriter(wm.op, store, deviceID, aw.filterSpec, wm.errchan)
```

Comment:
The error is also returned during any writes on the writer.  I see why you want to do this, but we serialize all writes to the writers so if there is an error, it gets returned in io.Copy() and we catch it there, log it, and return.  Have you seen a case where that isn't so?